### ISSUE TITLE
"ModuleNotFoundError: No module named 'tkinter'" in Windows 10

### ISSUE THREAD

**Issue by 2lu3** [pypa/pipenv#4712]
### Issue description

I'm Windows 10 user.
It seems that tkinter is not in pipenv virtual environment.

I ran this command in command prompt in windows terminal.
```
> pipenv install
> pipenv shell
(.venv) >python -c "import tkinter"
```

### Expected result

```
(blank line)
```

### Actual result

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'tkinter'
```

### Steps to replicate

```
> mkdir test
> cd test
> python --version
Python 3.9.5
> python -c "import tkinter"

> pipenv install
> pipenv shell
(.venv) > python --version
Python 3.9.5
(.venv) > where python
C:\Users\****\test\.venv\Scripts\python.exe
C:\Users\****\AppData\Local\Microsoft\WindowsApps\python.exe
(.venv) >python -c "import tkinter"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'tkinter'
```

-------------------------------------------------------------------------------

<details><summary>$ pipenv --support</summary>

Pipenv version: `'2020.11.15'`

Pipenv location: `'C:\\Users\\tlaloc\\AppData\\Local\\Packages\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\LocalCache\\local-packages\\Python39\\site-packages\\pipenv'`

Python location: `'C:\\Users\\tlaloc\\AppData\\Local\\Microsoft\\WindowsApps\\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\\python.exe'`

Python installations found:

  - `3.9.5`: `C:\Python39\python.exe`
  - `3.7.8`: `C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\python.exe`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.9.5',
 'os_name': 'nt',
 'platform_machine': 'AMD64',
 'platform_python_implementation': 'CPython',
 'platform_release': '10',
 'platform_system': 'Windows',
 'platform_version': '10.0.19042',
 'python_full_version': '3.9.5',
 'python_version': '3.9',
 'sys_platform': 'win32'}
```

System environment variables:

  - `ALLUSERSPROFILE`
  - `APPDATA`
  - `CHOCOLATEYINSTALL`
  - `CHOCOLATEYLASTPATHUPDATE`
  - `COMMONPROGRAMFILES`
  - `COMMONPROGRAMFILES(X86)`
  - `COMMONPROGRAMW6432`
  - `COMPUTERNAME`
  - `COMSPEC`
  - `CUDA_PATH`
  - `CUDA_PATH_V11_3`
  - `DRIVERDATA`
  - `HOMEDRIVE`
  - `HOMEPATH`
  - `LOCALAPPDATA`
  - `LOGONSERVER`
  - `NUMBER_OF_PROCESSORS`
  - `NVCUDASAMPLES11_3_ROOT`
  - `NVCUDASAMPLES_ROOT`
  - `NVTOOLSEXT_PATH`
  - `ONEDRIVE`
  - `ONEDRIVECONSUMER`
  - `OS`
  - `PATH`
  - `PATHEXT`
  - `PIPENV_ACTIVE`
  - `PIPENV_VENV_IN_PROJECT`
  - `PIP_DISABLE_PIP_VERSION_CHECK`
  - `PIP_PYTHON_PATH`
  - `PROCESSOR_ARCHITECTURE`
  - `PROCESSOR_IDENTIFIER`
  - `PROCESSOR_LEVEL`
  - `PROCESSOR_REVISION`
  - `PROGRAMDATA`
  - `PROGRAMFILES`
  - `PROGRAMFILES(X86)`
  - `PROGRAMW6432`
  - `PROMPT`
  - `PSMODULEPATH`
  - `PUBLIC`
  - `PYTHONDONTWRITEBYTECODE`
  - `PYTHONUSERBASE`
  - `SYSTEMDRIVE`
  - `SYSTEMROOT`
  - `TEMP`
  - `TMP`
  - `USERDOMAIN`
  - `USERDOMAIN_ROAMINGPROFILE`
  - `USERNAME`
  - `USERPROFILE`
  - `VBOX_HWVIRTEX_IGNORE_SVM_IN_USE`
  - `VIRTUAL_ENV`
  - `WINDIR`
  - `WSLENV`
  - `WT_PROFILE_ID`
  - `WT_SESSION`
  - `XDG_CONFIG_HOME`
  - `PIP_SHIMS_BASE_MODULE`
  - `PYTHONFINDER_IGNORE_UNSUPPORTED`

Pipenv窶都pecific environment variables:

 - `PIPENV_ACTIVE`: `1`
 - `PIPENV_VENV_IN_PROJECT`: `true`

Debug窶都pecific environment variables:

  - `PATH`: `C:\Users\tlaloc\programming\test\.venv\Scripts;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\bin;C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.3\libnvvp;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files\NVIDIA Corporation\NVIDIA NvDLISR;C:\Program Files\ffmpeg-4.4-full_build\bin;C:\Program Files\Neovim\bin;C:\Program Files\nodejs\;C:\ProgramData\chocolatey\bin;C:\Program Files (x86)\mingw-w64\i686-8.1.0-posix-dwarf-rt_v6-rev0\mingw32\bin;C:\Program Files\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin;C:\Program Files\Git\bin;C:\Program Files\NVIDIA Corporation\Nsight Compute 2021.1.1\;C:\Program Files\MATLAB\R2021a\bin;C:\Program Files\heroku\bin;C:\Program Files\PostgreSQL\13\bin;C:\Users\tlaloc\AppData\Local\Microsoft\WindowsApps;C:\Users\tlaloc\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\tlaloc\AppData\Local\Programs\Python\Python39\Scripts\;C:\Users\tlaloc\AppData\Local\Programs\Python\Python39\;C:\Users\tlaloc\AppData\Roaming\npm;C:\Program Files\Git\usr\local\bin;C:\Users\tlaloc\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\Scripts`
  - `VIRTUAL_ENV`: `C:\Users\tlaloc\programming\test\.venv`

---------------------------

Contents of `Pipfile` ('C:\\Users\\tlaloc\\programming\\test\\Pipfile'):

```toml
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]

[dev-packages]

[requires]
python_version = "3.9"

```

Contents of `Pipfile.lock` ('C:\\Users\\tlaloc\\programming\\test\\Pipfile.lock'):

```json
{
    "_meta": {
        "hash": {
            "sha256": "a36a5392bb1e8bbc06bfaa0761e52593cf2d83b486696bf54667ba8da616c839"
        },
        "pipfile-spec": 6,
        "requires": {
            "python_version": "3.9"
        },
        "sources": [
            {
                "name": "pypi",
                "url": "https://pypi.org/simple",
                "verify_ssl": true
            }
        ]
    },
    "default": {},
    "develop": {}
}

```
</details>

**Comment by niphadkarneha** [comment]
Does this stack article help: https://stackoverflow.com/questions/25905540/importerror-no-module-named-tkinter

**Comment by 2lu3** [comment]
Isn't this for Linux?
My environment is windows

**Comment by frostming** [comment]
Does it exist in host Python? Does it exist in a venv manually created by `virtualenv` by hand?

**Comment by 2lu3** [comment]
I ran this code, which creates virtualenv by hand. And there was no problem about tkinter.

> python -m venv .venv
> .venv\Scripts\activate.bat
 (.venv) >python
Python 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tkinter
>>> exit()

After that, I ran below code again, to make it clear that the problem is in pipenv. However, there was no problem. So I think reinstalling python may fix this problem.
> pipenv install
> pipenv shell
(.venv) >python -c "import tkinter"

So I will close this.
Thank you!
